gateway-modules/apim-ballerina-configs/configs/org/wso2/carbon/apimgt/gateway/utils/Utils.bal

```diff
@@ -20,6 +20,10 @@ function constructSubscriptionNotFound (message response) {
     json payload = {"code":900903, "message":"subscription not found"};
     messages:setJsonPayload(response, payload);
 }
+function constructAPIIsInMaintenance (message response) {
+    json payload = {"code":700700, "message":"This API has been blocked temporarily"};
+    messages:setJsonPayload(response, payload);
+}
```

Comment:
Fixed in https://github.com/wso2/carbon-apimgt/pull/4087/commits/e9aae8173ce0104f16c45795dee53026a884e801